docs/src/UserGuide/read.md

```diff
@@ -43,6 +43,9 @@ Individual arrays can be accessed using subsetting:
 ds.tos
 ````
 
+Please note that netCDF4 uses HDF5 which is not thread-safe in Julia.
+Use manual [locks](https://docs.julialang.org/en/v1/manual/multi-threading/#man-using-locks) in your own code to avoid any data-race.
```

Comment:
maybe could you put here, the how? the link is fine but having the solution here would be nice 😄 .

Reply:
I added an example in [c9f55a2](https://github.com/JuliaDataCubes/YAXArrays.jl/pull/451/commits/c9f55a2045123b6514740f03879aa9c3ba6e35fc). Simple enough for demonstration but a little bit silly. Is there any nice real-world example of this?

Reply:
mmm... it looks like https://github.com/JuliaIO/HDF5.jl/pull/1021  fixed this, weird that still fails here. However, is good to know how to address this if seeing the wild. Thanks!